docs/source/requirements/core.rst

```diff
@@ -200,7 +217,24 @@ Wachgänger deaktivieren
 Akzeptanzkriterien
 ~~~~~~~~~~~~~~~~~~
 
-*tbd*
+#. Als Wachleiter ist es möglich einen vorhandenen Wachgänger, welcher keine Dienste mehr übernimmt, zu deaktivieren. 
```

Comment:
@thomasscheuermann Soll das ein Wachleiter wirklich können, ich sehe das eher als Feature für den TL?

Reply:
In der Funktionsbeschreibung war dies bereits wie folgt hinterlegt:
_Als Wachleiter möchte ich einen bereits angelegten Wachgänger deaktivieren können, sofern dieser als Mitglied ausgeschieden ist oder aus anderen Gründen keine Wachdienste mehr übernimmt._

Gerne können wir diese Funktion auch den TLs vorbehalten.